Updating BCs Documentation (#250)
shannon-lab/zapdos: doc/content/source/bcs/NeumannCircuitVoltageMoles_KV.md

# NeumannCircuitVoltageMoles_KV

!syntax description /BCs/NeumannCircuitVoltageMoles_KV

## Overview

> `NeumannCircuitVoltageMoles_KV` is a Neumann boundary condition the equates the plasma and drive potential to the plasma current using Kirchoff’s voltage law for a circuit.

Reviewer: ```suggestion
`NeumannCircuitVoltageMoles_KV` is a Neumann boundary condition that equates the plasma and drive potential to the plasma current using Kirchoff’s voltage law for a circuit.
```

The Kirchoff’s voltage law at the cathode is defined as

\begin{equation}
V_\text{source} + V_\text{cathode} = e\left( \Gamma_{i} - \Gamma_{e}  \right) AR
\end{equation}

Where $V_\text{source}$ is the driven potential, $V_\text{cathode}$ is the potential at cathode,

> $e$ is the elemental charge , $\Gamma_i$ and $\Gamma_e$ are fluxes ions and electrons respectively, $A$ is the cross-sectional area of the plasma, and $R$ is the ballast resistance. Within `NeumannCircuitVoltageMoles_KV`, the charged species' flux at the boundary are assumed to be given by:

Reviewer: ```suggestion
$e$ is the elemental charge , $\Gamma_i$ and $\Gamma_e$ are the ion and and electron particle fluxes, respectively, $A$ is the cross-sectional area of the plasma, and $R$ is the ballast resistance. Within `NeumannCircuitVoltageMoles_KV`, the charged species' flux at the boundary is assumed to be given by:
```

\begin{equation}
a =
\begin{cases}

> 1, & \mu_{e} \left( \nabla (V) \right) \cdot \textbf{n} > 0\\

Author: Yes, the signs are wrapped up in the mobility coefficients. I edited any files that had two `a` coefficients to clarify the definition of `a` a little better in [0148c94](https://github.com/shannon-lab/zapdos/pull/250/commits/0148c946a64f82036d24677ca0945202e2241eb6).

$\mu$ is the mobility coefficient, $n$ is the species density, $v_\text{th}$ is the thermal velocity of the species, $n_{\varepsilon}$ is the mean energy density of electrons, $k_{B}$ is the Boltzmann constant, $T$ is the gas temperature, and $n_{\gamma}$ is electron density emitted by the surface. $a$ is defined such that the outflow is non-zero when the drift velocity is directed towards the wall and zero otherwise. $r$ is defined as the fraction of particles reflected by the surface.

To convert Kirchoff’s voltage law into a Neumann type BC, the set of equations are rearranged, so that this boundary condition solves for the potential flux normal for the boundary, such that:

\begin{equation}
\nabla (V) \cdot \textbf{n} =\frac{\frac{1+r}{1-r}\left( \text{-} V_\text{source}-V_\text{cathode} \right) + \frac{1}{2} e \left[ v_{\text{th},e}\left( n_{e} - n_{\gamma} \right) - \left[(1-a)\gamma - 1\right]v_{\text{th},i}n_{i} \right] \ AR}{e \ \left[ (2a-1)\mu_{e} \left( n_{e} - n_{\gamma} \right) - (2a_{i}-1)\left[(a-1)\gamma - 1\right]\mu_{i}n_{i} \right] AR}
\end{equation}

> When converting the density to logarithmic form and applying a scaling factor of the mesh and voltage, the strong form for `NeumannCircuitVoltageMoles_KV` is defined as

Reviewer: ```suggestion
When converting the density to logarithmic-molar form and applying a scaling factor of the mesh and voltage, the strong form for `NeumannCircuitVoltageMoles_KV` is defined as
```

> ## Overview

Author: @gsgall Reworded a few lines and defined subscripts that were used multiple times. Please let me know if that better clarified the documentation.
